apple.china.conf

```diff
@@ -42,6 +44,7 @@ server=/images.apple.com/114.114.114.114
 server=/init-p01md.apple.com/114.114.114.114
 server=/init-p01st.push.apple.com/114.114.114.114
 server=/init-s01st.push.apple.com/114.114.114.114
+server=/init.ess.apple.com/114.114.114.114
```

Comment:
This domain doesn't seem to have a China mainland node.

Reply:
Just wrote a new script and analyze the pcap file automatically. You are right. This domain shall not be included in the file and I'll remove it later.